the-collab-lab/tcl-56-smart-shopping-list
# Al dn welcome prompt

## Description



This code adds a welcome prompt that is displayed when a user's list is empty, inviting them to add an item to begin their list. We added an "Add Item" button above the filter for easier accessibility that redirects to the Add Item page. We also created a potato chip graphic (gif) to display the welcome message; and added a List.css file to position "Chippy".

## Related Issue


closes #7 

## Acceptance Criteria



 The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before


<img width="1432" alt="Screen Shot 2023-04-25 at 5 32 35 PM" src="https://user-images.githubusercontent.com/102399239/234436628-8c2979c3-6329-40e4-8ab7-7ee8d8875295.png">

### After



<img width="1410" alt="Screen Shot 2023-04-25 at 5 31 52 PM" src="https://user-images.githubusercontent.com/102399239/234436650-07712fb1-a740-4e96-b12b-4dbec37d4c49.png">


## Testing Steps / QA Criteria



When you create a new list in the home page, check to see if the welcome message is displayed. It should change to a different suggestion once you have added an item to your list.
